docs/sansio.rst

```diff
@@ -66,7 +66,7 @@ without requiring the use of the :class:`Event` class.
       The *headers* mapping is expected to support lowercase keys.
 
       Since this method assumes the body of the HTTP request is only of the
-      [content type that GitHub sends](https://developer.github.com/webhooks/creating/#content-type),
+      `content type that GitHub sends <https://developer.github.com/webhooks/creating/#content-type>`__),
```

Comment:
The `)` at the end can be removed, and there should only be one `_`.

Reply:
Thanks, got confused by the unhelpful underline style!